### DISCUSSION TITLE
Trigger listener once screen is loaded

### DISCUSSION THREAD

**Discussion by mark-austen-charterhouse** [orchidsoftware/platform#1550] [Q&A]
Is there a way to trigger a listener once the screen is loaded?

I have a simple form in which the user enters a stock code. The listener looks up the stock name and supplier and displays them. These values are not stored in the the table.

This works well for creating a new bill, but when the user edits the bill, the stock name and supplier are not visible. Or not until the stock code is changed in some way.

Thanks.

**Comment by ZarcoNontol** [comment]
I would like to know if there is a way to do it as well! Thanks

**Reply by MarkAusten** [reply]
Best I've found for now is a standard jQuery script that runs on document ready that triggers the change event of the control. There doesn't seem to be a way to do this in the Orchid framework. Not yet, at least.

**Comment by tabuna** [comment]
Initially, we reflect the listener layer just like the others.
You can render content on the server-side without making an additional request if you can't do it. 
Can you give me an example that I can look at?
